4-Experimental-Accepted/RFCXXXX-Snap-Preview.md

```diff
@@ -0,0 +1,55 @@
+---
+RFC: RFC
+Author: Travis Plunk (@TravisEz13)
+Status: Experimental-Accepted
+Version: 0.1
+Area: Packaging
+Comments Due: June 29, 2018
+Plan to implement: Yes
+---
+
+# Snaps of Preview builds
+
+We currently have not published any snap packages of a preview build.  There are some significant questions before we do.
```

Comment:
You should link to https://snapcraft.io/ for people not familiar with Snap.  Also consistently capitalize `Snap`.  Finally, each sentence should be on a separate line in a markdown file (semantic line feeds).